bindings/python/CMakeLists.txt

```diff
@@ -36,6 +36,10 @@ SET(PYWRAP ${PROJECT_NAME}_pywrap)
 SET(PYWRAP ${PYWRAP} PARENT_SCOPE)
 
 # --- DEPENDENCIES --- #
+find_package(Eigen3 QUIET)
```

Comment:
Eigen3 is already included by the main CMakeLists.txt.
These lines are then not necessary.

Reply:
>     2\. Search of Eigen3 in quiet mode using CMake logic - the original logic and pkg-config steps are still preserved. The reason for this is to be able to pass in the path of `eigenpy` directly in the cmake configuration step which would stop the inclusion of the eigenpy dependency eigen3 which is not otherwise resolved.

@wxmerkt I certainly do not understand well why you have to perform this step. Can you rephrase it?

Reply:
Thanks for this clarification.
I just have a remaining question, if `PKG_CONFIG_PATH`variable is not available in Catkin, how do you manage the dependencies?

Reply:
We specify them in the `package.xml` and thus they will get installed automatically during the `rosdep install` (urdfdom, boost-python). For eigenpy, we pass it in directly. The rest is available automatically. We run Travis on Kinetic and Melodic and it passes well.